packages/url_launcher/url_launcher_linux/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.0-nullsafety.1
```

Comment:
I just finished https://github.com/flutter/plugins/pull/3584 and only belatedly realized it will cause conflicts with this. To simplify things, I've folded the `implements` change into that PR, so you can revert the url_launcher parts of this.

Reply:
ack